exercises/04_power/power.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import "fmt"
+
+func Pow(x, y uint) uint {
+	if y == 0 {
+		x = 1
+		return x // because all situation, when y = 0 -> x = 1. Test without it doesn't work (by x^y)
+	}
```

Comment:
You can get rid of this `if` block, but you will need to do some changes to the for loop.